test/integration/patients/patient/patient-flow.js

```diff
@@ -228,6 +243,38 @@ context('patient flow page', function() {
       .get('.sidebar')
       .find('[data-duration-region]')
       .should('contain', '20 mins');
+
+    cy.sendWs({
+      category: 'CommentAdded',
+      resource: {
+        type: 'patient-actions',
+        id: testFlowAction.id,
+      },
+      payload: {
+        comment: {
+          type: 'comments',
+          id: uuid(),
+        },
+        attributes: {
+          message: 'New websocket comment.',
+          created_at: testCreatedAt,
+          edited_at: null,
+        },
+        relationships: {
+          _clinician: testClinician.id,
+        },
+      },
+    });
```

Comment:
I'm not sure how this socket message data should be structured. But as long as we have the following, we should be able to just add the comment to the activity collection without doing an api request:

* Comment `id`
* Comment `message`
* Comment `created_at`
* Comment `edited_at`
* Author/clinician `id`

Socket message data is currently looks like this: https://github.com/RoundingWell/care-ops-backend/blob/develop/tests/Notifications/Data/Messages/comment-added-message-structure.json

Current ideas/questions:

1. Could we just assume `edited_at` is always `null` for this? Since that will be `null` whenever a new comment is created.
2. Do we want a web socket notification for when a comment has been edited?
3. We could just assume `created_at` is always `now()`? If that's not included in the socket message data. Can there be a delay behind the scenes between the comment being created and the socket notification being emitted?

Reply:
hmm this is getting complicated.  Can we move on to other stuff for a bit?